src/Select.jsx

```diff
@@ -14,6 +14,7 @@ import {
   getPropValue,
   getValuePropValue,
   isCombobox,
+  isMultiple,
```

Comment:
I think tag should use the same logic. so should use isMultipleOrTags here.

Reply:
There's some problem about tags here. As a user, when I use 
* 'multiple' Select:  input only used as filter. It's good and not that annoying to always keep filter there.
* 'tags' Select, inputs can be transformed to a selected item. If just update 'isMultiple' to 'isMultipleOrTags', after a random input and select, input was kept there to filter out only the added item. It's definitely not what users want.

Reply:
So we may
* update 'multiple' for now
* add some prop/config as suggested in https://github.com/ant-design/ant-design/issues/9573
* or make tags smarter - keep filter only when the input value is in previous select options

What do you think?

Reply:
I think add a props `keepSearchText` is enough. What's your option @afc163 @yesmeck

Reply:
Yes, we need to introduce a new prop to enable this behavior.

Reply:
Maybe `keepInputFilter: 'always' | 'no' | 'smart'` ? I prefer the `smart keep/clear search input` way on `tags` though it's not that easy to implement.  And, as there're already lots of Select props, still tend not to add more.

Leave further work to you @yutingzhao1991 .

Reply:
I think `keepSearchInput: true | false` is enough.

Reply:
react-component/select@7531b4d

Add a prop of `autoClearSearchValue`, default `false`.